department-of-veterans-affairs/vets-website
# Update explanation for reasonForCurrentSeparation

## Summary

On the Reason for separation page, updates the deprecated field with the v3 component equivalent.

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/76774
(Issue notes the `reasonForSeparation` field on this page, but that's a radio field which already uses a web component. I interpreted the issue as referencing the `otherExplanation` field which is nested within the `reasonForSeparation` field, and was deprecated.)

## Testing done

- Set the marriage status as "Separated" and continue until reaching the "Reason for Separation" page.
- Select the "Other" option and fill in the input. Confirm that the entered text is added to Redux. Continue to the review page.
- Confirm that the entered text appears in the review page.

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Desktop |    <img width="564" alt="Screenshot 2024-03-06 at 9 41 16 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/e6568924-6c73-42fe-81ab-bd0c0aa20a5f">    |    <img width="564" alt="Screenshot 2024-03-06 at 9 51 39 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/e7f1d36c-53b6-4a82-9430-81ea2c1b3387">   |

## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria

https://github.com/department-of-veterans-affairs/va.gov-team/issues/76774

### Quality Assurance & Testing


### Error Handling


### Authentication
